packages/vanilla-bundle/src/components/slick-vanilla-grid-bundle.ts

```diff
@@ -5,7 +5,6 @@ import 'slickgrid/slick.core';
 import 'slickgrid/slick.grid';
 import 'slickgrid/slick.dataview';
 import 'slickgrid/slick.groupitemmetadataprovider';
-import 'slickgrid/plugins/slick.resizer';
```

Comment:
this is no longer used, it should have been removed in previous PR from last week